components/prism-falcon.js

```diff
@@ -0,0 +1,26 @@
+Prism.languages.falcon = Prism.languages.extend('javascript', {
+	'number': [
+		Prism.languages.javascript.number,
+		{
+			pattern: /\b(?:api)\b/
+		}
+	],
+	'builtin': /\b(?:get|post|put|delete|head|byte|int|int8|int16|int32|int64|uint|uint8|uint64|int_t|int8_t|int16_t|int32_t|int64_t|uint_t|uint8_t|uint64_t|long|Long|float|Float|double|Double|boolean|Boolean|bool|Bool|string|String|binary|string|Function|any|number|boolean|Array|symbol|console|Promise|unknown|never)\b/,
+});
+
+Prism.languages.falcon['class-name'][0].pattern = /(\b(?:enum|struct|class|interface|extends|implements|instanceof|new)\s+)[\w.\\]+/;
```

Comment:
Please sort the keyword list.
```suggestion
Prism.languages.falcon['class-name'][0].pattern = /(\b(?:class|enum|extends|implements|instanceof|interface|new|struct)\s+)[\w.\\]+/;
```